src/main/webapp/js/userMapData.json

```diff
@@ -926,25 +1072,36 @@
     "Stephen F. Austin State University",
     "Stevens Institute of Technology",
     "Sun Valley High School",
+    "SUNY Buffalo",
     "SUNY Jefferson Community College",
     "Syracuse University",
     "Tarleton State University",
     "Tarrant County College",
+    "Taylor University",
     "Temple University",
     "Tennessee Tech University",
+    "Tennessee Technological University",
```

Comment:
This is the same as the entry preceding this, but since the one spelled in full seems more appropriate, you can delete the entry before.